src/sass/pages/career.scss

```diff
@@ -2,4 +2,208 @@
     .hero__picture img {
         margin-bottom: -25px;
     }
+
+    article.band__container {
+        @include row;
+        @include container;
+        margin-bottom: $margin-bottom-careers-section;
+    }
+
+    h1.striked + div {
+        margin-top: $margin-top-careers-content;
+    }
+
+    .offers {
+        &__list {
+            margin-top: $margin-top-careers-content;
+        }
+    }
+
+    .text-section {
+        @include columns(8);
+        @include offset(2);
+
+        ul {
+            margin-top: $gutter-size;
+            columns: 2;
+
+            li {
+                padding-top: $gutter-size / 2;
+                padding-left: $padding-left-careers-benefits-list-item;
+                position: relative;
+
+                &:before {
+                    content: " ";
+                    display: block;
+                    position: absolute;
+                    left: 0;
+                    top: $size-careers-benefits-list-item-dot-spacing-top;
+                    background-color: $color-careers-benefits-dot;
+                    height: $size-careers-benefits-list-item-dot;
+                    width: $size-careers-benefits-list-item-dot;
+                    border-radius: 50%;
+                }
+            }
+        }
+    }
+
+    .hq-gallery {
```

Comment:
since the `hq-gallery` block has no custom style, I'd lower 1 indentation level doing:
```.hq-gallery__item {```